### ISSUE TITLE
Do you know Revopoint POP and Revopoint POP 2?

### ISSUE THREAD

**Issue by huminlian** [zjudmd1015/Mini-3D-Scanner#9]
3D scanners became more and more popular in the world as some consumer-grade [3D scanners](https://shop.revopoint3d.com/) were released. The price of these 3D scanners is affordable for personal users. Revopoint POP is the most famous 3D scanner of these products. The upgrade version of Revopoint POP—Revopoint POP 2 has risen more than $ 2.61 million on Kickstarter so far.

**Comment by miachelbrown** [comment]
Yes, I'm familiar with the Revopoint POP series. The Revopoint POP series represents a line of [3D scanners ](https://www.revopoint3d.com/) developed by Revopoint, a company known for its innovative scanning solutions. The series has seen notable advancements, with each iteration building upon the capabilities of its predecessor.
The Revopoint POP 2, for example, was a significant improvement over the original POP model. It introduced enhanced scanning accuracy, faster processing speeds, and improved compatibility with various devices. This allowed users to capture intricate details with greater precision, making it a valuable tool for various applications.
However, the most recent addition to the POP series is the Revopoint POP 3, which was released in June 2023. This latest model promises even more impressive features and capabilities. With its advanced technology, portability, and user-friendly interface, the POP 3 continues the tradition of delivering high-quality 3D scanning experiences.
The Revopoint POP series has undoubtedly evolved to cater to the growing demands of users across various industries, providing efficient and effective solutions for 3D scanning needs. The introduction of the POP 3 reflects Revopoint's commitment to innovation and improving the user experience in the realm of 3D scanning technology.
